homeassistant/components/fritz/common.py

```diff
@@ -220,7 +220,7 @@ async def _async_update_data(self) -> None:
         """Update FritzboxTools data."""
         try:
             await self.async_scan_devices()
-        except (FritzSecurityError, FritzConnectionException) as ex:
+        except FRITZ_EXCEPTIONS as ex:
             raise update_coordinator.UpdateFailed from ex
```

Comment:
```suggestion
            raise update_coordinator.UpdateFailed("blah blah blah") from ex
```

We should add some more info on what actually caused the failure.
Current log message:

`2022-02-03 08:56:25 ERROR (MainThread) [homeassistant.components.fritz.common] Error fetching fritz-192.168.188.1-coordinator data:`